packages/docs/docs/advanced/custom-components.mdx

```diff
@@ -0,0 +1,141 @@
+---
+slug: /custom-components
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+import CodeBlock from '@theme/CodeBlock';
+import source from '!!raw-loader!@motion-canvas/examples/src/scenes/components-switch';
+
+# Components
+
+<AnimationPlayer name="components" banner />
+
+Components are reusable parts of code such as `Rect` and `Circle` which can be
```

Comment:
Additionally: did we want these to link to the API pages for `Rect` and `Circle`?

Reply:
yes we do